lib/deimos/config/configuration.rb

```diff
@@ -481,6 +481,10 @@ def self.configure_producer_or_consumer(kafka_config)
       setting :published_state
       # Value to set the state_column to if publishing fails - state-based only.
       setting :failed_state
+
+      # Inherited state based poller class name to use for publishing
```

Comment:
Actually it shouldn't need to be state based - any subclass should work.

Reply:
@dorner Sounds good. I guess in terms of the time_based poller class, I would also need to make the same changes I did to the state_based poller class (i.e. updated the fetch_results method)?